packages/terra-date-input/src/DateInput.jsx

```diff
@@ -5,15 +5,24 @@ import classNamesBind from 'classnames/bind';
 import ThemeContext from 'terra-theme-context';
 import uuidv4 from 'uuid/v4';
 import { injectIntl } from 'react-intl';
-import Input from 'terra-form-input';
```

Comment:
Since it no longer needs terra-form-input and this appears to be the only place that imports it, can it now be removed as a dependency?

Reply:
I think yes. I'll try it.

Reply:
Actually, I must leave it in because my test harness (doc tests) require this input.